src/m_config.c

```diff
@@ -1021,6 +1021,13 @@ static default_t extra_defaults_list[] =
 
     CONFIG_VARIABLE_FLOAT(fsynth_reverb_width),
 
+    //!
+    // Fine tune the FluidSynth output level. Default is 1.0,
+    // range is 0.1 - 10.0.
```

Comment:
Why this specific range?

Reply:
It's the same range we use in Woof, but it's arbitrary. I'm pretty sure `10.0` will seriously harm your ears, though. Maybe we should limit it to the `0.5..10.0` range.

Reply:
`10` is a maximum value for [fluid_synth_set_gain()](https://www.fluidsynth.org/api/group__synthesis__params.html#gab1929557284fdfeb59abf63bde83d35c) function.

Reply:
That method goes down to zero. Should we allow that? Or some really small number?

Reply:
We probably should, very loud soundfonts exist too. I think `0` is simpler and more intuitive.